libraries/c_sdk_v4/integration-test/core_mqtt_system_test.c

```diff
@@ -55,6 +55,9 @@
 /* Include clock for timer. */
 #include "platform/iot_clock.h"
```

Comment:
Do we need the include for iot_clock.h?

Reply:
Got the `IotClock_GetTimeMs` function that is passed to `MQTT_Init`

Reply:
Can we get rid of that dependency as well? As these tests are for FreeRTOS, using FreeRTOS functions would be ideal. An extra layer of abstraction may not required. We will be able to reuse the getTime function that you come up with in demos as well.

Reply:
I can do the hygiene in a separate PR.